pymilvus/client/prepare.py

```diff
@@ -559,14 +463,17 @@ def search_requests_with_expr(cls, collection_name, data, anns_field, param, lim
             "ignore_growing": ignore_growing,
         }
 
+        if anns_field:
+            search_params["anns_field"] = anns_field
+
```

Comment:
anns_field can be None?

Reply:
yes, server will handle this. proxy will deduce it from schema.
see https://github.com/milvus-io/milvus/pull/24499